operating_procedures.md

```diff
@@ -62,33 +60,42 @@ In collaboration with Editors and Maintainers, Contributors help drive the evolu
 
   FG Maintainers are responsible for ensuring that the contents of documents and work products accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. 
 
-  FG Chair(s) will designate a FG Maintainer or FG Maintainers for the FG. FG Maintainer(s) are overseen by of the FG Chair(s).
+  FG Chair(s) designates any FG Maintainer(s) for the FG. FG Maintainers are overseen by the FG Chair(s). Maintainer designations will be revisited at the discretion of the FG chair, typically at the conclusion of a release cycle.  
+
+  To be designated as a Maintainer, individuals MUST:
 
-  The FG Maintainer(s) MUST keep the project documentation up to date (e.g.: contributing, readme and release planning documents).
+   * Be involved as a Contributor consistently over at least one release cycle
+   * Lead or significantly contribute technical content to a major Issue or PR 
+   * Regularly participate in the Review & Approval process, including meeting discussions, issue/PR comments, asynchronous discourse (e.g. Slack), and PR approvals
+   * Continue these activities through each release cycle
 
-  The FG Maintainer(s) MUST apply the "Review & Approval" process—covered in 6.1 of this document—to contributions submitted by the Working Group members.
+  The FG Maintainer(s) MUST keep the project documentation up to date (e.g.: CONTRIBUTING, README and RELEASE PLANNING documents), and MUST have a good working knowledge of GitHub and other Project tools in use.
+
+  The FG Maintainer(s) MUST apply the "Review & Approval" process-covered in this document-to contributions submitted by the Working Group members.
+
+  > Editor Note: Add link to Review & Approval section after adding "FOCUS Work Flow Procedures"
 
   The FG Maintainer(s) SHOULD use GitHub Labels to indicate the type of “Review & Approval” assigned to each Pull Request.
 
-  The FG Maintainer(s) SHOULD keep FOCUS project related communications with members  within GitHub using GitHub Issues or GitHub Pull Requests and corresponding GitHub comment functionality rather than one to one communications.
+  The FG Maintainer(s) SHOULD keep FOCUS project related communications with members  within GitHub using GitHub Issues or GitHub Pull Requests and corresponding GitHub comment functionality rather than one-to-one communications.
 
-  The FG Maintainer(s) SHOULD close contributions that do not follow the rules, or meet the right quality or are related to features that are in the scope of the Release Version under development.
+  The FG Maintainer(s) MAY suggest to close contributions that do not follow the rules, fail to meet the required quality standards, or are unrelated to the features in the scope of the Release Version under development.
 
```

Comment:
Each Github Issue in the scope of the Release Version under development SHOULD be sponsored by at least one FG Maintainer.  At least one of the Maintainer sponsors of an Issue MUST:

- Assemble a team to consider and development the Issue
- Provide technical guidance and oversight for development
- Consistently attend the recurring Task Force meetings to which the Issue has been assigned
- Provide progress updates in recurring Task Force meetings
- Help unblock technical hurdles encountered during development
- Ensure the timely resolution of the Issue, based on the release planning timeline